docs/release-notes/9.0.asciidoc

```diff
@@ -4,29 +4,65 @@
 coming::[9.0.0]
 
 [discrete]
-[[release-notes-9.0.0]]
-=== 9.0.0
+[[release-notes-9.0.0-beta]]
+=== 9.0.0-beta
 
 [discrete]
-[[known-issue-9.0.0]]
+[[breaking-changes-9.0.0-beta]]
+==== Breaking changes
+* Removes deprecated bulk action API endpoints ({kibana-pull}207906[#207906]).
+* Refactors the Timeline HTTP API endpoints ({kibana-pull}200633[#200633]).
+* Removes deprecated API endpoints for {elastic-defend} ({kibana-pull}199598[#199598]).
+* Disables deprecated API endpoints for bulk CRUD actions ({kibana-pull}197422[#197422]).
+
+[discrete]
+[[deprecations-9.0.0-beta]]
+==== Deprecations
+* Renames the `integration-assistant` plugin to `automatic-import` to match the associated feature ({kibana-pull}207325[#207325]).
+* Removes all legacy risk engine code and features ({kibana-pull}201810[#201810]).
+* Removes deprecated API endpoints for {elastic-defend} ({kibana-pull}199598[#199598]).
+* Deprecates the SIEM signals migration APIs ({kibana-pull}202662[#202662]). 
+
+[discrete]
+[[known-issue-9.0.0-beta]]
 ==== Known issues
 
+// tag::known-issue[]
 [discrete]
-[[breaking-changes-9.0.0]]
-==== Breaking changes
+.Duplicate alerts can be produced from manually running threshold rules 
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running threshold rules could produce duplicate alerts if the date range was already covered by a scheduled rule execution.
+====
+// end::known-issue[]
 
+// tag::known-issue[]
 [discrete]
-[[deprecations-9.0.0]]
-==== Deprecations
+.Manually running custom query rules with suppression could suppress more alerts than expected
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running a custom query rule with suppression could incorrectly inflate the number of suppressed alerts. 
+====
+// end::known-issue[]
 
 [discrete]
-[[features-9.0.0]]
+[[features-9.0.0-beta]]
 ==== New features
+* Enables Automatic Import to accept CEL log samples ({kibana-pull}206491[#206491]).
+* Applies the latest Elastic UI framework (EUI) to {elastic-sec} features ({kibana-pull}204007[#204007], {kibana-pull}204908[#204908]).
```

Comment:
should this PR be in features? https://github.com/elastic/kibana/pull/203320

Reply:
I can add it in. The release notes tool is only retrieving this for the 8.18 release notes, which is why it was missing from this draft. Thanks for checking!

Reply:
```suggestion
* Applies the latest Elastic UI framework (EUI) to {elastic-sec} features ({kibana-pull}204007[#204007], {kibana-pull}204908[#204908]).
* Allows you to view {es} queries that run during rule execution. Adds the option for threshold, custom query, and {ml} rules ({kibana-pull}203320[#203320]). 
```